finagle-core/src/main/scala/com/twitter/finagle/util/LoadService.scala

```diff
@@ -21,11 +21,18 @@ import scala.io.Source
  */
 private object ClassPath {
 
-  private val ignoredPackages = Seq(
+  private[util] var ignoredPackages = Seq(
```

Comment:
instead of making this mutable and adding a method in LoadService to update it, let's just add ignoredPaths directly when we declare ignoredPackages.